chain/chunks/src/test/multi.rs

```diff
@@ -98,9 +100,11 @@ fn basic_setup(config: BasicSetupConfig) -> ShardsManagerTestLoop {
         test.register_handler(forward_network_request_to_shards_manager().widen().for_index(idx));
         test.register_handler(capture_events::<ShardsManagerResponse>().widen().for_index(idx));
         test.register_handler(route_shards_manager_network_messages(NETWORK_DELAY));
-        // Note that we don't have the periodically resending requests handler, because
-        // our forwarding logic means that we don't need to resend requests, unless
-        // there is unreliable network, which is tested separately.
+        test.register_handler(
+            periodically_resend_chunk_requests(time::Duration::milliseconds(400))
```

Comment:
Could we make this duration a constant like `CHUNK_REQUEST_DELAY` and reuse below?

Reply:
Fixed.